### PR TITLE
Added example for using Optuna

### PR DESCRIPTION
**Pull request by sampreet-arthi**
It's only an example and I haven't added proper dependencies and classes etc.

### CODE REVIEW

File: genrl/deep/hyperparameters/optuna/a2c_cartpole-v0.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import numpy as np
```
Comment (Sharad24) [inline genrl/deep/hyperparameters/optuna/a2c_cartpole-v0.py:1]: Delete the binary file?

**Review by Sharad24 — COMMENTED** [review]
Can you fetch upstream and merge into your master? :)

Also, should we have this in examples rather than part of genrl?

**Comment by Sharad24** [comment]
Or maybe add it after #256 and to the utils folder

File: genrl/deep/hyperparameters/optuna/a2c_cartpole-v0.py (hunk 1 above)
Comment (sampreet-arthi) [reply genrl/deep/hyperparameters/optuna/a2c_cartpole-v0.py]: I've kept it there for usage. It can be loaded with `read.py`.

**Comment by sampreet-arthi** [comment]
> Or maybe add it after #256 and to the utils folder

Ok cool. Makes sense

**Comment by Sharad24** [comment]
Resolve the LGTM alerts before merging

**Comment by sampreet-arthi** [comment]
Should I also add some files with working hyperparameters?

**Comment by sampreet-arthi** [comment]
Solves #250. I could address #249 too here.

**Comment by Sharad24** [comment]
Lets merge this. We can raise a separate PR for another issue.

**Pull request merged**